### ISSUE TITLE
Validate my admin screen for accessibility

### ISSUE THREAD

**Issue by dartiss** [dartiss/no-self-ping#4]
https://achecker.ca/checker/index.php

**Comment by dartiss** [comment]
### Known Problems

**Check 95**: [textarea element missing an associated label.](https://achecker.ca/checker/suggestion.php?id=95)

**Repair**: Add a label element immediately before or after the textarea element. Set the for attribute value of the label element to the same value as the id attribute value of the textarea element. Add label text to the label element. Or, set the title attribute value to the textarea element to the label text. Or, add a label element that surrounds the textarea element and add label text.

**Line 2, Column 276**:
<textarea name="no_self_pings_option" rows="3" class="large-text code"></textarea>

**Comment by dartiss** [comment]
There are some potential problems too. These can be seen by running the following code through the above checker...

```
<h2>No Self Pings</h2>
By default, No Self Pings will exclude pings for this site (https://artiss.blog) but you can supply additional URLs below. Separate multiple URLs with line breaks.<table class="form-table"><tr><th scope="row"><label for="no_self_pings_option">Additional URLs</label></th><td><textarea name="no_self_pings_option" rows="3" class="large-text code"></textarea></td></tr></table>
```

**Comment by dartiss** [comment]
Fixed in #6
